packages/esm-patient-chart-app/src/clinical-views/encounter-list/encounter-list.component.tsx

```diff
@@ -6,12 +6,11 @@ import { EncounterListDataTable } from './table.component';
 import { Button, Link, OverflowMenu, OverflowMenuItem, DataTableSkeleton, Pagination } from '@carbon/react';
 import { Add } from '@carbon/react/icons';
 import { launchEncounterForm } from '../utils/helpers';
-import { deleteEncounter } from '../encounter-list.resource';
+import { deleteEncounter } from '../utils/encounter-list.resource';
 import { useEncounterRows, useFormsJson } from '../hooks';
 
 import styles from './encounter-list.scss';
-import { type TableRow, type Encounter, type Mode, type ColumnValue } from '../types';
-import { type FormattedColumn } from '../utils/encounter-list-config-builder';
+import { type TableRow, type Encounter, type Mode, type ColumnValue, type FormattedColumn } from '../types';
```

Comment:
Can we reorder the imports here and elsewhere to conform to [our code organization standards](https://o3-docs.openmrs.org/docs/coding-conventions/code-organization), please?

Reply:
Done